### PR TITLE
INTYGFV-14887: UeCauseOfDeath and UeCauseOfDeathList components

### PR DESCRIPTION
**Pull request by SopraErik**
UeCauseOfDeath is  used causes of death. One question per cause.
UeCauseOfDeathList is used for the list of other diseases that contributed (https://inera.atlassian.net/browse/INTYGFV-14867, https://inera.atlassian.net/browse/INTYGFV-14892 and https://inera.atlassian.net/browse/INTYGFV-14893)

### CODE REVIEW

File: packages/common/src/components/Inputs/Dropdown.tsx [hunk 1]
```diff
@@ -10,20 +11,30 @@ interface Props {
...
       {label !== 'undefined' ? <label htmlFor={id}>{label}</label> : null}
       <div
+        css={heightStyle}
         className={classNames('ic-forms__select', {
           'iu-border-error dropdown': hasValidationError,
           dropdown: !hasValidationError,
           'ic-forms__select--disabled': disabled,
         })}>
-        <select value={value} name={name} id={id} disabled={disabled} onChange={onChange}>
+        <select css={heightStyle} value={value} name={name} id={id} disabled={disabled} onChange={onChange}>
```
Comment (SamuelTilly) [inline packages/common/src/components/Inputs/Dropdown.tsx:37]: Use styled component instead of CSS properties.

**Review by SamuelTilly — CHANGES_REQUESTED** [review]

File: packages/common/src/utils/faker/fakeCertificateData.ts [hunk 2]
```diff
@@ -682,3 +687,112 @@ export const fakeHeaderElement = (
...
+        type: ConfigTypes.UE_CAUSE_OF_DEATH_LIST,
+        list: [
+          {
+            id: 'A',
+            type: ConfigTypes.UE_CAUSE_OF_DEATH,
+            label: 'A',
+            specifications: [
+              { id: 'UPPGIFT_SAKNAS', label: 'Uppgift saknas' },
+              { id: 'KRONISK', label: 'Kronisk' },
+              { id: 'PLOTSLIG', label: 'Plötslig' },
```
[the remark this answers is not in the document]
Comment (SopraErik) [reply packages/common/src/utils/faker/fakeCertificateData.ts:739]: So probably AKUT I guess, this needs to be synced with backend @mww-sopra-steria @ChrilleSigerhed

File: packages/common/src/components/Inputs/Dropdown.tsx [hunk 3]
```diff
@@ -10,20 +11,30 @@ interface Props {
...
 
 const Dropdown: React.FC<Props> = (props) => {
   const { onChange, label, name, id, value, hasValidationError, options, disabled } = props
+
+  const heightStyle = props.height
+    ? css`
+        height: ${props.height} !important;
+        min-height: ${props.height} !important;
+      `
+    : css``
```
[the remark this answers is not in the document]
Comment (SopraErik) [reply packages/common/src/components/Inputs/Dropdown.tsx]: yes ,definitely, seems to be due to some application specific modifications to inera css in index.css

[the remark this answers is not in the document]
Comment (SopraErik) [reply packages/common/src/components/Inputs/Dropdown.tsx]: I have created a bug now: 
[https://inera.atlassian.net/browse/INTYGFV-14970](https://inera.atlassian.net/browse/INTYGFV-14970)

**Comment by SopraErik** [comment]
Significant changes made to this now since the PR originally was created, as the data model changed. This PR now also handles additional diseases and the ability to add more rows. @ChrilleSigerhed , @mww-sopra-steria , @SamuelTilly , @HelenaEdblad @UlrikaSopra , please review it again

**Comment by SopraErik** [comment]
Converting this into draft, as there are still some issues, such as deletion of rows, that needs to be solved

**Comment by SopraErik** [comment]
Thanks @SamuelTilly , nice work!

**Pull request merged**